### ISSUE TITLE
[Bug]: game crash

### ISSUE THREAD

**Issue by mindryder** [bonsaistudi0s/Enderman-Overhaul#17]
### Bug Description

Mod crash game

### How to Reproduce?

[crash-2023-10-17_19.40.02-client.txt](https://github.com/bonsaistudi0s/Enderman-Overhaul/files/12936765/crash-2023-10-17_19.40.02-client.txt)

### Version

1.0.1

### Mod Loader Version

1.20.1 

### Mod Loader

Fabric

### Code of Conduct

- [X] I have searched the issue tracker and confirmed that the issue has not been reported
- [X] I have checked the FAQ (if one is present) and my issue is not listed
- [X] I have verified that I am using the latest version of the mod
- [X] I have verified that I have installed all the required dependencies for the mod
- [X] I have verified that I do not have Optifine installed. We will close issues if we detect Optifine is in your pack. We cannot fix Optifine issues. Please do not report issues with Optifine present.

**Comment by morceaudebois** [comment]
I'm having the same crash here. To add some detail, I killed a few endermen in a warped forest in the Nether, then died so I respawned in the overworld. It crashed, and now whenever I log into that world, it crashes a few seconds after spawning.

Still works on the same instance in another world though, so it seems that something got broken in the initial world. 

My log: https://mclo.gs/sAifOBy
